.github/workflows/kit-bootstrap.yml

```diff
@@ -0,0 +1,60 @@
+name: This workflow automates the build, testing, and bootstrapping of a test environment on Kubernetes using KIT CLI
+on:
+  workflow_run:
+    workflows: [Trigger-kit]
+    types: [completed]
+permissions:
+  id-token: write
+  pull-requests: write
+  contents: read
+jobs:
+  kit:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Why do we checkout twice? There is another checkout in `install-go-and-dependencies`?

Reply:
Its Necessary to checkout twice ,Otherwise `Error: Can't find 'action.yml', 'action.yaml' or 'Dockerfile' under '/home/runner/work/kubernetes-iteration-toolkit/kubernetes-iteration-toolkit/.github/actions/install-go-and-dependencies'. Did you forget to run actions/checkout before running your local action?` throws an error ,Tested locally with and without `- uses: actions/checkout@v2`